geo_activity_playground/core/activities.py

```diff
@@ -89,6 +90,19 @@ def get_time_series(self, id: int) -> pd.DataFrame:
                     * 3.6
                 )
                 changed = True
+            else:
```

Comment:
This is a nice idea, but in #82 there came the suggestion to use the `units` field from the FIT files. I think that makes more sense to use the actual data instead of a heuristic.

Reply:
Yes, I do agree. Shall I rebase on the updated master which includes PR #82 ?

Reply:
I don't think that rebasing makes sense. I've already manually cherry-picked some of your ideas into the `master` branch. If you want to want to make further changes I'd suggest that you base them on the latest `master`. And perhaps let me know what you plan, then I can perhaps already implement it based on your idea.